### PR TITLE
alias for generators, rich option

### PR DESCRIPTION
**Pull request by yonatanmn**
in this pull request:
1. alias for generators : `a`,`c`,`s` (action,store,component).
2. `rich` flag when generating a component.
3. protection against empty name in action/store generator (prev resulted :`UndefinedStore`)  
4. updated readme and description in package.json (that's what displayed in yeoman's site)

### CODE REVIEW

**Comment by newtriks** [comment]
@yonatanmn thanks for this, could you look at the conflicts please?

**Comment by yonatanmn** [comment]
Where can I see the conflicts in github site?

File: component/index.js [hunk 1]
```diff
@@ -10,8 +10,13 @@ util.inherits(ComponentGenerator, ScriptBase);
 
 ComponentGenerator.prototype.createComponentFile = function createComponentFile() {
   this.option('es6');
+  this.option('rich');
 
   this.es6 = this.options.es6;
+  this.rich = this.options.rich;
+  console.log(this.architecture)
```
Comment (ColCh) [inline component/index.js]: Please remove `console.log`

File: component/index.js [hunk 2]
```diff
@@ -10,8 +10,13 @@ util.inherits(ComponentGenerator, ScriptBase);
...
+
+  console.log('Creating a component');
```
Comment (ColCh) [inline component/index.js]: Please remove `console.log`, also this one

Comment (yonatanmn) [inline component/index.js]: Done

Comment (ColCh) [inline component/index.js]: Nice!

**Comment by yonatanmn** [comment]
Finally had time to merge...

**Comment by newtriks** [comment]
Thanks @yonatanmn

**Pull request merged**